Data/future/usa/margins/NG.csv

```diff
@@ -64,3 +63,52 @@ date,initial,maintenance
 20150729,2035,1850
 20160223,2090,1900
 20160411,1815,1650
+20160630,1980,1800
+20160708,2255,2050
+20160730,2420,2200
+20160831,2585,2350
+20161206,2805,2550
+20161214,3135,2850
+20161223,3410,3100
+20170224,2860,2600
+20170301,2420,2200
+20170405,2200,2000
+20170429,2090,1900
+20170711,1925,1750
+20170729,1760,1600
+20170831,1980,1800
+20171102,2090,1900
+20171115,2420,2200
+20180224,1815,1650
+20180302,1705,1550
+20180308,1457.5,1325
+20180428,1320,1200
+20180920,1485,1350
+20180928,1705,1550
+20181002,1980,1800
+20181006,2145,1950
+20181107,2475,2250
+20181110,2695,2450
+20181114,3080,2800
+20181119,4070,3700
+20181117,5170,4700
+20181123,6270,5700
+20190104,5060,4600
+20190108,3960,3600
+20190115,4620,4200
+20190118,5060,4600
+20190205,3960,3600
+20190212,3025,2750
+20190226,2035,1850
+20190404,1815,1650
+20190501,1540,1400
+20190628,1650,1500
+20190709,1815,1650
+20190709,1815,1650
+20190911,1980,1800
```

Comment:
While NG was present in this advisory, there was a previous advisory that updated NG to these values on `2019-09-06` - https://www.cmegroup.com/notices/clearing/2019/09/Chadv19-293.html